NASA-AMMOS/aerie-ui
# Dev sequencing

Depends on: https://github.com/NASA-AMMOS/aerie/pull/1444

Easiest rebase of my life? I was expecting this to be a lot worse. Needs thorough testing and review.

Adds the new sequence editor and a major refactor of how we are using dictionaries. All dictionaries are now part of a parcel that needs to be assigned to items that previously pointed directly to a command dictionary.

To Test:
* Create a new parcel that contains some different dictionaries.
* Navigate to the sequence editor page
* Create a new sequence using the new grammar

Everything else should be working nominally.
